e2e/visual_regression/import/import_page.test.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  loginAsVisualRegressionUser2,
+  webuiEndpoint,
+} from '../../utils/test-util';
+import { expect, test } from '@playwright/test';
+
+test.describe('Import & Run page Visual Regression Test', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.setViewportSize({
+      width: 1500,
+      height: 1500,
+    });
+    await loginAsVisualRegressionUser2(page);
+    await page.getByRole('menuitem', { name: 'Import & Run' }).click();
+  });
```

Comment:
we don't need to have to log in and go to the data page for every test. Can we leave it outside of test.describe?



```suggestion
  test.beforeEach(async ({ page }) => {
    await page.setViewportSize({
      width: 1500,
      height: 1500,
    });
    await loginAsVisualRegressionUser2(page);
    await page.getByRole('menuitem', { name: 'Import & Run' }).click();
  });
test.describe('Import & Run page Visual Regression Test', () => {

```